apps/react-18-tests-v8/tsconfig.react-18.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "target": "ES2019",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": ["ES2019", "dom"],
+    "sourceMap": true,
+    "strict": true,
```

Comment:
discussed offline: 

Although v8 is not using strict mode, we should test against it and relax checking only where a big effort would be needed to mitigate theses issues within implementation of v8. 

This approach also mirrors what our users have in their codebase to have better coverage